### ISSUE TITLE
No some/every functions

### ISSUE THREAD

**Issue by treyhunner** [cmstead/JFP#1]
Examples:
- http://underscorejs.org/#some
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

**Comment by cmstead** [comment]
I added this to the next round of to-dos. Look for contains and every functions.

**Comment by cmstead** [comment]
Contains and every functions are now available in the v1.2.0 branch build. Though this is an upcoming release, the functions are fully tested and should be useable in code now.

**Comment by cmstead** [comment]
The 1.2.0 version is available.  I just wanted to follow up and make sure it was known. : )

**Comment by treyhunner** [comment]
I saw the release.  Thanks for the update though.

I just tried using `every` for my detect-charset package instead of lodash's every but it didn't work because yours doesn't pass in an index.

Also, I don't see a 'some' function.  Did that not make the cut?

**Comment by cmstead** [comment]
Good catch. I'll add an index.  The some is 'contains' instead.

**Comment by treyhunner** [comment]
`some` doesn't mean the same thing as `contains` to me.

Underscore's [contains](http://underscorejs.org/#contains) is different than [some](http://underscorejs.org/#some).

`some` is also what the ES5 spec uses.

**Comment by cmstead** [comment]
I'm more sold on the difference between some and contains as defined by ES5 as well as languages such as Lisp and Haskell than lodash/underscore as I'm not looking to create a drop-in for a different library.

That said, I typically look at these functions as mathematical constructs and so I view it as either 'all' or 'there exists a' so some and contains are mathematically interchangeable ~all constructs.  I should have taken a look at the ES spec.

TL;DR

Okay. One sec and I'll have it up.

**Comment by cmstead** [comment]
I just pushed an update that should resolve the missing some function as well as a redefinition of contains.  You should also now see an index available to your predicate functions for both every and some.  This is available in v1.2.1

https://github.com/cmstead/JFP/releases/tag/v1.2.1
